src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -40,7 +36,7 @@ class CardsResource extends Resource {
         new ResourceObject(
                 id: card.id,
                 type: 'card',
-                attributes: new SimpleCard (
+                attributes: new Card (
```

Comment:
I think you don't need to create a new card object here. You could so something like `attributes: card`